### PR TITLE
High contrast documentation

### PR DESCRIPTION
**Pull request by Guy-Moses**

### CODE REVIEW

File: Tutorial/HighContrastSupport.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+# Accessibility: Adding High-Contrast Mode Support
+Windows *High-Contrast* setting makes text and apps easier to see by using more distinct colors.
+Read more about [high-contrast support in Power BI](https://powerbi.microsoft.com/en-us/blog/power-bi-desktop-june-2018-feature-summary/#highContrast).
+
+Adding high-contrast support to your visual requires the following:
+1. On init: Detect whether Power BI is in high-contrast mode and if so, get current high-contrast colors.
+2. Every update: Change the way the visual renders to make it easier to see.
+
+## On Init
+The [colorPalette](./ColorPalette.md) member of `options.host` has several properties for high-contrast mode. Use these properties to determine whether high-contrast mode is active, and if yes, what colors to use.
```
Comment (AviSander) [inline Tutorial/HighContrastSupport.md]: 'and if so'

File: Tutorial/HighContrastSupport.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+            this.foregroundColor = colorPalette.foreground.value;
+            this.backgroundColor = colorPalette.background.value;
+            this.foregroundSelectedColor = colorPalette.foregroundSelected.value;
+            this.hyperlinkColor = colorPalette.hyperlink.value;
+        }
+```
+Alternatively, you can store the `host` object during initialization and access the relevant `colorPalette` properties during update.
+
+## On Update
+How exactly you draw your visual in high-contrast mode depends on your visual's specific graphic design. Usually, a somewhat different design is required to keep the important details easy to distinguish with the limited colors.
```
Comment (AviSander) [inline Tutorial/HighContrastSupport.md]: I'd rephrase this:
Each visual would typically design its own implementation...

File: Tutorial/HighContrastSupport.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+* For slicers, active filter elements use foreground-selected color.
+
+In Sample Bar Chart, for example, all bars are drawn with 2 pixels thick foreground outline and background fill. Compare the way it looks with default colors and with a couple of high-contrast themes:
+
+![Sample Bar Chart using standard colors](images/HC_sampleBarChart_standard.png)
+
+![Sample Bar Chart using *Dark #2* color theme](images/HC_sampleBarChart_dark2.png)
+![Sample Bar Chart using *White* color theme](images/HC_sampleBarChart_white.png)
+
+Here is one place in `visualTransform` function that was changed to support high-contrast, it is called as part of rendering during `update`:
```
Comment (AviSander) [inline Tutorial/HighContrastSupport.md]: ...place in THE ...

File: Tutorial/HighContrastSupport.md (hunk 1 above)
Comment (Guy-Moses) [reply Tutorial/HighContrastSupport.md]: done

File: Tutorial/HighContrastSupport.md (hunk 3 above)
Comment (Guy-Moses) [reply Tutorial/HighContrastSupport.md]: done

File: Tutorial/HighContrastSupport.md (hunk 2 above)
Comment (Guy-Moses) [reply Tutorial/HighContrastSupport.md]: Done with rephrasing (a visual doesn't design its implementation, the devs [should] do that)
